voice/session.go

```diff
@@ -37,26 +37,21 @@ var WSTimeout = 10 * time.Second
 // Session is a single voice session that wraps around the voice gateway and UDP
 // connection.
 type Session struct {
+	// TODO: expose getters mutex-guarded.
```

Comment:
This comment doesn't belong here, `Handler` is already exposed. Add it above `gateway` and `voiceUDP` again.